tests/selectors/home.selectors.ts

```diff
@@ -10,4 +10,6 @@ export enum HomePageSelectors {
   BalancesTabBtn = 'tab-balances',
   SwapBtn = 'swap-btn',
   FundAccountBtn = 'fund-account-btn',
+  BalancesList = 'balances-list',
+  BtnFundAccount = 'btn-fund-account',
```

Comment:
Shouldn't this be `FundAccountBtn` to follow pattern here?

Reply:
removed entirely was copied from old tests